### DISCUSSION TITLE
x-pixel co-ordinates offset by 1 pixel (fixed: change the clkphase value)

### DISCUSSION THREAD

**Discussion by evil-oz** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#523] [Q&A] [answered]
am using 4 standard 64x32 panels chained, with Brian Lough's shield.

Everything is steady and super fast, but there is a 1 pixel horizontal shift. There should be 2 green lines between panel 1 and 2.
(note that the corner led of the panel 4 is dead, so ignore that missing pixel)

Did almost no modifications to ChainedPanels demo, except for panel order ( set to CHAIN_TOP_LEFT_DOWN )

Tried also with different order and also using directly MatrixPanel_I2S_DMA instead of Virtual display.

What am I doing wrong?

(was using same panels with 8266 for years with no issues)

thanks
![IMG_20231125_135247](https://github.com/mrfaptastic/ESP32-HUB75-MatrixPanel-DMA/assets/16673184/89f29934-d0f8-447c-9759-5f15b5945ccb)
![IMG_20231125_135338](https://github.com/mrfaptastic/ESP32-HUB75-MatrixPanel-DMA/assets/16673184/75a3f78f-297e-4eef-aa90-b259b83fdf48)

**Comment by davemaster** [comment]
all panels, at the back side, align the "up" arrows to the same direction, the "right" arrows, too

**Reply by evil-oz** [reply]
thought i have to put them as drawn below?

![image](https://github.com/mrfaptastic/ESP32-HUB75-MatrixPanel-DMA/assets/16673184/7b30dd81-2096-4876-b32e-fed9a1d85850)

**Comment by mrcodetastic** [comment, marked as answer]
clkphase config value needs to be changed?

Please refer to common issues: https://github.com/mrfaptastic/ESP32-HUB75-MatrixPanel-DMA/issues/134

**Reply by evil-oz** [reply]
woah, that was it, moved it to false et voila

big thanks!

![image](https://github.com/mrfaptastic/ESP32-HUB75-MatrixPanel-DMA/assets/16673184/3ea6e086-a720-4b5e-b3ed-a32a6b9aecc1)
